samples/dashboards/responsive/responsive-breakpoints/demo.js

```diff
@@ -0,0 +1,89 @@
+const csvData = document.getElementById('csv').innerText;
+
+Dashboards.board('container', {
+    editMode: {
+        enabled: true,
+        contextMenu: {
+            enabled: true
+        }
+    },
+    dataPool: {
+        connectors: [
+            {
+                id: 'sample',
+                type: 'CSV',
+                options: {
+                    csv: csvData,
+                    firstRowAsNames: true
+                }
+            }
+        ]
+    },
+    gui: {
+        layouts: [
+            {
+                id: 'layout-1',
+                rows: [
+                    {
+                        cells: [
+                            {
+                                id: 'dashboard-col-0'
+                            },
+                            {
+                                id: 'dashboard-col-1'
+                            }
+                        ]
+                    }
+                ]
+            }
+        ]
+    },
+    responsiveBreakpoints: {
+        small: 300,
+        medium: 500,
+        large: 600
```

Comment:
I do not see an effect:
<img width="434" alt="Screenshot 2023-06-28 at 14 59 19" src="https://github.com/highcharts/highcharts/assets/40056287/487e04d7-56ac-4cf7-89ec-43234fccab46">

Reply:
You need to enable the edit mode and click one of the responsive buttons, perhaps we should describe that?

Reply:
Oh! I agree. A description in source or details would be helpful.